lib/github-service.js

```diff
@@ -29,6 +37,23 @@ class GithubService {
     });
   }
 
+  async createPullRequestComment(report){
```

Comment:
async keyword should be removed since there is no await now.

Reply:
Done

Reply:
Async keyword still there ?